### PR TITLE
:recycle: replace turbo by mocha test runner

### PR DESCRIPTION
**Pull request by corinnekrych**

### CODE REVIEW

**Comment by corinnekrych** [comment]
@danielpassos for subtask https://issues.jboss.org/browse/RHMAP-14690

File: README.md [hunk 1]
```diff
@@ -6,6 +6,30 @@ This is a blank 'hello world' RHMAP MBaaS, demonstrating the UnifiedPush Server
...
 'Hello world' endpoint, that triggers a push notification requests against RHMAP.
+
+# Build
+```
+npm install
+```
+
+# Tests
+
+All the tests are in the "test/" directory. The cloud app is using moka as the test runner.
```
Comment (danielpassos) [inline README.md]: s/moka/mocha?

File: README.md [hunk 2]
```diff
@@ -6,6 +6,30 @@ This is a blank 'hello world' RHMAP MBaaS, demonstrating the UnifiedPush Server
...
+```
+* acceptance tests
+```    
+npm run serve
+npm run accept
+```
+* coverage report for unit tests:
+```
+npm run coverage-unit
+```
```
Comment (danielpassos) [inline README.md]: There is no `coverage-accept`?

**Comment by corinnekrych** [comment]
@danielpassos Do you mind having a second 👁  ?

**Pull request merged**